src/Temporalio/Bridge/include/temporal-sdk-bridge.h

```diff
@@ -106,6 +106,7 @@ typedef struct ClientOptions {
   const struct ClientTlsOptions *tls_options;
   const struct ClientRetryOptions *retry_options;
   const struct ClientKeepAliveOptions *keep_alive_options;
+  const HttpConnectProxyOptions *http_connect_proxy_options;
```

Comment:
This is auto-generated as part of the Rust build and therefore should match the Rust constructs

Reply:
I'm getting an inconsistent experience, but this is ultimately what is gen'd locally for me.

Reply:
No prob, I may have just assumed it was going to generate `const struct`. If CI passes w/ what is generated here we're all good. Can ignore.